.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "samverschueren.final-newline",
+    "DmitryDorofeev.empty-indent",
```

Comment:
I believe `prettier` takes care of removing whitespace lines and adding a final newline.

Reply:
Yeah, I think we can defer to Prettier for these 2.